app/code/community/Bolt/Boltpay/Helper/Bugsnag.php

```diff
@@ -163,4 +163,11 @@ protected function getBoltTraceId()
 
         return null;
     }
+
+    
+    public function setMetaData(array $metaData)
```

Comment:
`set` sounds like it would override existing value. can you rename it to `addMetaData`?

Reply:
@daisy1754 addMetaData already used for breadcrumb , so i rename to addBoltMetaData